examples/example-gateway/endpoints/serverless/serverless-empty.yaml

```diff
@@ -0,0 +1,15 @@
+clientId:
```

Comment:
add a tchannel endpointType example to see this would work.

Reply:
This will be done in a follow up diff.